### ISSUE TITLE
./plexupdate.sh -p -C produces non-actionable output

### ISSUE THREAD

**Issue by pjwelsh** [mrworf/plexupdate#65]
first run:
./plexupdate.sh -p -C
Your distribution may require the use of the AUTOSTART [-s] option for the service to start after the upgrade completes.
Downloading release "plexmediaserver-1.0.0.2261-a17e99e.i386.rpm"...OK

second run:.
/plexupdate.sh -p -C
Your distribution may require the use of the AUTOSTART [-s] option for the service to start after the upgrade completes.

There is also a exit 2 as the return status.
Expected output should be nothing and return code of 0.
The AUTOSTART exit 2 should likely be a function of either AUTOINSTALL or AUTOUPDATE set to "yes" instead.
Basically creates the option for using this script as a cron checker for updates that only notifies when there is an update.

Thanks
PJ

**Comment by mrworf** [comment]
The reason it returns 2 is so you know what happened. The cron option only hides additional outputs, it will not change basic behavior of the script. This is however not the first comment on this, so I think I need to rethink this option a bit :)

**Comment by pjwelsh** [comment]
Thank you very much for looking into this!
PJ

July 16 2016 3:44 PM, "Henric Andersson"  wrote:
	Closed #65 (https://github.com/mrworf/plexupdate/issues/65) via f3f24fc (https://github.com/mrworf/plexupdate/commit/f3f24fc97efcb62b1dada4e6f8a66c6b8f6bcfad).
